src/server.ts

```diff
@@ -23,32 +26,49 @@ const swaggerSpec = swaggerJsDoc(options);
 app.use('/api/docs', swaggerUI.serve, swaggerUI.setup(swaggerSpec));
 
 app.use('/api', routes);
+app.use('/docs', swaggerUI.serve, swaggerUI.setup(swaggerSpec));
```

Comment:
remove this please, why did you add it there, yet it is alread on line 26 of this file